docs/03.API-EXAMPLE.md

```diff
@@ -1,8 +1,42 @@
 JerryScript Engine can be embedded into any application, providing the way to run JavaScript in a large range of environments - from desktops to low-memory microcontrollers.
 
-This guide is intended to introduce you to JerryScript embedding API through creation of simple JavaScript shell.
+This guide is intended to introduce you to JerryScript embedding API and to create a minimal JavaScript shell.
+The examples are not using all API methods please also check out the API reference document which contains additional examples.
 
-## Step 1. Execute JavaScript from your application
+
+## Before trying out the examples: Get and build JerryScript library
+
+Before getting started using the JerryScript library it should be cloned and built for a target os/device.
+
+There are quite a few configuration options but for these examples the JerryScript is built
+**with default configuration** and installed to a user directory on a Linux system.
+This is done by the following commands:
+
+```sh
+$ mkdir jerry
+$ git clone https://github.com/jerryscript-project/jerryscript.git
+$ jerryscript/tools/build.py --builddir=$(pwd)/example_build --cmake-param="-DCMAKE_INSTALL_PREFIX=$(pwd)/example_install/"
+$ make -C $(pwd)/example_build install
+```
+
+With this the JerryScript library is installed into the `$(pwd)/example_install/{include,lib}` directories.
+
+In order to ease the usage of headers and libraries (archives) the `pkg-config` will be used and
+for it to detect the JerryScript files the following export is done (which is executed in the `jerry` directory):
```

Comment:
How about: "In this guide we will use `pkg-config` to ease the usage of headers and libraries. In order to do so, the following export is required (executed from the `jerry` directory)"?